source/TimePicker.js

```diff
@@ -75,13 +122,13 @@
 		* @private
 		*/
 		setupItem: function (inSender, inEvent) {
-			var index = inEvent.index % this.range;
-			this.$.item.setContent(this.meridiems[index]);
+			var index = inEvent.index % this.range || 0;
```

Comment:
Ya, I remember that it could be null at starting time but can't remember exact situation.
Let me find history.